### ISSUE TITLE
Export lottie json without fill?

### ISSUE THREAD

**Issue by badtant** [airbnb/lottie-web#1487]
Hi,

Is it possible to export a lottie json from AE without fill attributes so that the fill is black by default?
I want this to be able to set the fill with css without selecting any specific nodes inside the rendered svg.

Thanks

**Comment by bodymovin** [comment]
Hi, what do you mean without selecting specific nodes? How would you set the fill via css?

**Comment by badtant** [comment]
Or is it possible to get it to render fill="currentcolor"?

**Comment by bodymovin** [comment]
can you do?
`svg path {
            fill: rgb(255,0,0);
        }`

**Comment by badtant** [comment]
yes, that works.
but i'd rather see that i don't have to select any elements inside the svg :)

**Comment by bodymovin** [comment]
unfortunately, exporting without a fill is not possible.
You can try exporting with a stroke and using important to see if it works.

**Comment by badtant** [comment]
ok, then I'll settle with selecting path for now.
maybe exporting with currentColor can be something for the future

**Comment by markadrake** [comment]
Found this issue when searching for how to use `currentColor` in a lottie animation. The icon should not have a color defined for fill or stroke, it makes it very painful to change the color for any purpose. 

I've got a hamburger menu that needs to be black in it's default state, and white during it's active state.

I'm currently writing CSS to override inline props which is bad practice.

When looking at the JSON, how are colors determined? Would the script break if the colors were removed all together?
